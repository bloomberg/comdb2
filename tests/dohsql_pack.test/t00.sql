create table t1(x int)$$
insert into t1 values (1)

SELECT 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127 UNION ALL SELECT x, x+1, x+2, x+3, x+4, x+5, x+6, x+7, x+8, x+9, x+10, x+11, x+12, x+13, x+14, x+15, x+16, x+17, x+18, x+19, x+20, x+21, x+22, x+23, x+24, x+25, x+26, x+27, x+28, x+29, x+30, x+31, x+32, x+33, x+34, x+35, x+36, x+37, x+38, x+39, x+40, x+41, x+42, x+43, x+44, x+45, x+46, x+47, x+48, x+49, x+50, x+51, x+52, x+53, x+54, x+55, x+56, x+57, x+58, x+59, x+60, x+61, x+62, x+63, x+64, x+65, x+66, x+67, x+68, x+69, x+70, x+71, x+72, x+73, x+74, x+75, x+76, x+77, x+78, x+79, x+80, x+81, x+82, x+83, x+84, x+85, x+86, x+87, x+88, x+89, x+90, x+91, x+92, x+93, x+94, x+95, x+96, x+97, x+98, x+99, x+100, x+101, x+102, x+103, x+104, x+105, x+106, x+107, x+108, x+109, x+110, x+111, x+112, x+113, x+114, x+115, x+116, x+117, x+118, x+119, x+120, x+121, x+122, x+123, x+124, x+125, x+126 FROM t1
