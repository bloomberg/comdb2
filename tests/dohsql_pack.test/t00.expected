(rows inserted=1)
(1=1, 2=2, 3=3, 4=4, 5=5, 6=6, 7=7, 8=8, 9=9, 10=10, 11=11, 12=12, 13=13, 14=14, 15=15, 16=16, 17=17, 18=18, 19=19, 20=20, 21=21, 22=22, 23=23, 24=24, 25=25, 26=26, 27=27, 28=28, 29=29, 30=30, 31=31, 32=32, 33=33, 34=34, 35=35, 36=36, 37=37, 38=38, 39=39, 40=40, 41=41, 42=42, 43=43, 44=44, 45=45, 46=46, 47=47, 48=48, 49=49, 50=50, 51=51, 52=52, 53=53, 54=54, 55=55, 56=56, 57=57, 58=58, 59=59, 60=60, 61=61, 62=62, 63=63, 64=64, 65=65, 66=66, 67=67, 68=68, 69=69, 70=70, 71=71, 72=72, 73=73, 74=74, 75=75, 76=76, 77=77, 78=78, 79=79, 80=80, 81=81, 82=82, 83=83, 84=84, 85=85, 86=86, 87=87, 88=88, 89=89, 90=90, 91=91, 92=92, 93=93, 94=94, 95=95, 96=96, 97=97, 98=98, 99=99, 100=100, 101=101, 102=102, 103=103, 104=104, 105=105, 106=106, 107=107, 108=108, 109=109, 110=110, 111=111, 112=112, 113=113, 114=114, 115=115, 116=116, 117=117, 118=118, 119=119, 120=120, 121=121, 122=122, 123=123, 124=124, 125=125, 126=126, 127=127)
(1=1, 2=2, 3=3, 4=4, 5=5, 6=6, 7=7, 8=8, 9=9, 10=10, 11=11, 12=12, 13=13, 14=14, 15=15, 16=16, 17=17, 18=18, 19=19, 20=20, 21=21, 22=22, 23=23, 24=24, 25=25, 26=26, 27=27, 28=28, 29=29, 30=30, 31=31, 32=32, 33=33, 34=34, 35=35, 36=36, 37=37, 38=38, 39=39, 40=40, 41=41, 42=42, 43=43, 44=44, 45=45, 46=46, 47=47, 48=48, 49=49, 50=50, 51=51, 52=52, 53=53, 54=54, 55=55, 56=56, 57=57, 58=58, 59=59, 60=60, 61=61, 62=62, 63=63, 64=64, 65=65, 66=66, 67=67, 68=68, 69=69, 70=70, 71=71, 72=72, 73=73, 74=74, 75=75, 76=76, 77=77, 78=78, 79=79, 80=80, 81=81, 82=82, 83=83, 84=84, 85=85, 86=86, 87=87, 88=88, 89=89, 90=90, 91=91, 92=92, 93=93, 94=94, 95=95, 96=96, 97=97, 98=98, 99=99, 100=100, 101=101, 102=102, 103=103, 104=104, 105=105, 106=106, 107=107, 108=108, 109=109, 110=110, 111=111, 112=112, 113=113, 114=114, 115=115, 116=116, 117=117, 118=118, 119=119, 120=120, 121=121, 122=122, 123=123, 124=124, 125=125, 126=126, 127=127)
